### DISCUSSION TITLE
Will SWR capture the source repository of harvested records which are harvested from a secondary source?

### DISCUSSION THREAD

**Discussion by pvgenuchten** [soilwise-he/harvesters#50] [Q&A]
If a repository A is harvested by SWR, and knowing that that repository harvests itself a repository B, will the filters « Repository A and Repository B »  automatically be  displayed on the graphical interface of SWR ?

In other words, once a source repository is harvested, do the secondary source repositories are harvested too ?

**Comment by pvgenuchten** [comment]
Records harvested from OpenAire include information from which platform they are harvested (by openaire)
SWR captures this information and stores it.

Other repositories don't advertise where they harvested metadata from, however SWR may still detect that a record is available in multiple repositories (duplicities). SWR will capture those cases.

Soilwise uses the `DCTERMS.isReferencedBy` property to understand the souce repository, so you can advertise such history by providing that property.

**Comment by blitzfrayret** [comment]
If secondary repositories are harvested too, does it have to be mentioned in the term of use of the secondary source repositories ?

**Reply by pvgenuchten** [reply]
maybe this question relates to #61 ?

**Comment by blitzfrayret** [comment]
Would it be convenient to have a list of all the source repositories harvested by SWR (primary and secondary) ?

**Comment by pvgenuchten** [comment]
@blitzfrayret SWR does not harvest secondary repositories directly (for now). SWR does harvest records from a primary source, which itself has harvested content from a secondary source (for example we harvest european inspire portal, not from member states directly). 

--- 

I remember a case in INSPIRE in Poland where the national portal references a number of regional portals, the expectation was that the central harvest drills down into the regional portals to fetch additional content. Currently SWR does not apply this case.

**Reply by blitzfrayret** [reply]
So even if SWR does not harvest directly secondary repositories, is there a risk to harvest content (from secondary repositories) that would not come from European projects ?

**Reply by pvgenuchten** [reply]
This case is always available (primary of secondary), but is it a risk? The (non european) resource can still have interesting information?

**Comment by pvgenuchten** [comment]
A list of repositories harvested is currently not available, it is likely to change.
Notice that the list needs to be combined with a filter, because some portals we only harvest subsets.

You can see a list of harvested repositories and their filters in the [harvester CI folder](https://github.com/soilwise-he/harvesters/tree/main/CI)

**Comment by blitzfrayret** [comment]
Yes indeed. I have identified the task 1.5 'governance' leaded by Panos Ilias, I am going to forward him this issue, thank you. 


De: "Paul van Genuchten" ***@***.***> 
À: "soilwise-he/harvesters" ***@***.***> 
Cc: "Céline BLITZ-FRAYRET" ***@***.***>, "Mention" ***@***.***> 
Envoyé: Mardi 8 Octobre 2024 16:19:03 
Objet: Re: [soilwise-he/harvesters] Will SWR capture the source repository of harvested records which are harvested from a secondary source? (Discussion #50) 





maybe this question relates to [ https://github.com/soilwise-he/harvesters/issues/61 | #61 ] ?
